13.5.функции_с_возвратом_значения_часть2/13.5.8.py

```diff
@@ -0,0 +1,21 @@
+def isPrime(n):
+    if n % 2 == 0: return(n == 2)
+    d = 3
+    while d * d <= n and n % d != 0: d += 2
+    return(d * d > n)
+
+def isPalindrom(n):
+    n = str(n)
+    return(n == n[::-1])
+
+def isEven(n): return(not n % 2)
+
+def is_valid_password(password):
+    try:
+        a, b, c = map(int, password.split(':'))
+        return(isPalindrom(a) and isPrime(b) and isEven(c))
+    except: return(False)
```

Comment:
(По нюансам ниже кажется, что эту задачу ты копипастнул)

1) `return` не нужно писать на уровне заголовка функции
2) `try / except` 0_0? Это мы не проходили, это нам не задавали, парам-пам-пам
3) `return` - это не функция, ее не нужно вызывать со скобочками и передавать аргументы
4) по PEP8 имена переменных и функций должны быть в `snake_case`:
```suggestion
def is_prime(n):
    if n % 2 == 0: return(n == 2)
    d = 3
    while d * d <= n and n % d != 0: d += 2
    return d * d > n

def is_palindrom(n):
    n = str(n)
    return n == n[::-1]

def is_even(n): 
    return not n % 2

def is_valid_password(password):
    try:
        a, b, c = map(int, password.split(':'))
        return is_palindrom(a) and is_prime(b) and is_even(c)
    except: 
        return False
```